addons/godot-next/godot_next_plugin.gd

```diff
@@ -1,13 +1,17 @@
 tool
 extends EditorPlugin
+class_name MyPlugin
```

Comment:
If anything, I would call this GDNextPlugin, not MyPlugin. :-P

Reply:
sorry!! it was from a test!!!